.github/workflows/.windows.yml

```diff
@@ -15,7 +15,7 @@ on:
         default: false
 
 env:
-  GO_VERSION: "1.20.8"
+  GO_VERSION: "1.21.1"
```

Comment:
Why demote?

Reply:
it's 1.20 -> 1.21 :) (not 1.20.8 -> 1.20.1)

Reply:
😂 no worries! I saw your comment, and already thought you probably mis-read